core/Location.ml

```diff
@@ -32,7 +32,7 @@ let locate_start_end data s_pos e_pos =
 let locate ?(loc = NoSource) data = { data; loc }
 let empty pos = match pos with NoSource -> true | Source _ -> false
 
-let pp_column_range pos =
```

Comment:
Please revert the changes in Location.ml and Location.mli, this PR is already large without those.